cypress/support/appSettings.js

```diff
@@ -25,11 +25,27 @@ export function saveAppSettings(settings) {
 
 export function updateCybersourceConfiguration(orderSuffix = '') {
   it('Update cybersource app settings', () => {
+    cy.qe(`
+    GetApp settings query -
```

Comment:
For this use cy.addGraphqlLogs command
This would be available once this PR been merged - https://github.com/vtex-apps/cy-runner/pull/249/files